index.js

```diff
@@ -428,4 +438,4 @@ function plugin(options, Parser) {
       }
     }
   };
-};
+}
```

Comment:
Can you please return this semicolon?
```suggestion
};
```

Reply:
If you want me to, sure, but that's a function statement, right?

Reply:
Oh, I thought it was the end of `module.exports`. You're right.